manifest.json

```diff
@@ -6,8 +6,7 @@
   "homepage_url": "https://dnsimple.com",
   "description": "Always know if it resolves awesome",
   "permissions": [
-    "history",
-    "https://isitdnsimple.com/"
```

Comment:
You can't remove this, as this allows the extensions to make requests to that domain (which is used to determine the status).

Reply:
Are you sure about that? it's working good locally for me w/o that. Also I was not able to find that in the docs https://developer.chrome.com/apps/declare_permissions

Reply:
Yes 👉  https://developer.chrome.com/extensions/xhr#requesting-permission

Your docs link to ChromeOS

Reply:
You're totally right!